client/src/components/Amenities/Amenities.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import style from './Amenities.module.css';
+
+const myAmenities = (props) => {
+    return (
+      <>
```

Comment:
why empty component when you already have a top level div?

Reply:
If I put div, wouldn't extra node might come which is not required?